src/content/code/code.md

```diff
@@ -586,6 +586,48 @@ val query = graphql"{ hello }"
 Executor.execute(schema, query) map println
 ```
 
+#### [Caliban](https://ghostdogpr.github.io/caliban/) ([github](https://github.com/ghostdogpr/caliban)): Caliban is a purely functional library for building GraphQL servers and clients in Scala
+
+A simple example of a GraphQL schema and query with \`caliban\`:
```

Comment:
I don't love the word `simple` here (here's a [good talk from Jim Fisher](https://www.youtube.com/watch?v=gsT2BBWBVmM) that explains why). Maybe instead, it could briefly summarize what's happening in the schema/query?